src/Syntax/Trans/Scope.hs

```diff
@@ -35,9 +38,15 @@ newtype ScopeT a t f b = ScopeT (t f (Var a (f b)))
 unScopeT :: ScopeT a t f b -> t f (Var a (f b))
 unScopeT (ScopeT s) = s
 
+instance HFoldable t => HFoldable (ScopeT a t) where
+  hfoldMap f = getAlt . foldMap (Alt . f) <=< hfoldMap f . unScopeT
+
 instance (HFunctor t, forall g . Functor g => Functor (t g)) => HFunctor (ScopeT a t) where
   hmap f = ScopeT . hmap f . fmap (fmap f) . unScopeT
 
+instance HTraversable t => HTraversable (ScopeT a t) where
+  htraverse f = fmap ScopeT . htraverse f <=< traverse (traverse f) . unScopeT
+
```

Comment:
The symmetry here is even more apparent given the Kleisli composition in the `HFoldable` instance.